release-notes/opensearch-cli.release-notes.1.1.0.md

```diff
@@ -0,0 +1,20 @@
+## Version 1.1.0 Release Notes
+
+### Enhancements
+
+* Fix config file permission ([#29](https://github.com/opensearch-project/opensearch-cli/pull/29))
```

Comment:
Is this a bug fix or an enhancement?

Reply:
Updated title to reflect this as enhancement.